src/content/it/2020/http2.md

```diff
@@ -0,0 +1,684 @@
+
+E la ripartizione della pagina stessa? In genere parliamo della differenza tra contenuti di prima parte (o proprietari) e di terze parti. Terza parte è definita come contenuto non sotto il controllo diretto del proprietario del sito, che fornisce funzionalità come pubblicità, marketing o analisi. La definizione di terze parti nota è tratta dal repository <a hreflang="en" href="https://github.com/patrickhulce/third-party-web/blob/8afa2d8cadddec8f0db39e7d715c07e85fb0f8ec/data/entities.json5">web di terze parti</a>.
+
+La Figura 22.7 ordina ogni sito Web per la frazione di richieste HTTP/2 per terze parti note o richieste di prima parte rispetto ad altre richieste. C'è una differenza notevole in quanto oltre il 40% di tutti i siti non ha richieste HTTP/2 o gQUIC di prima parte. Al contrario, anche il 5% più basso delle pagine ha il 30% di contenuti di terze parti serviti su HTTP/2. Ciò indica che gran parte dell'ampia adozione di HTTP/2 è guidata da terze parti.
+
+{{ figure_markup(
+  image="http2-first-and-third-party-http2-usage.png",
+  caption="La distribuzione della frazione di richieste HTTP/2 di terze parti e di prima parte per pagina.",
+  description="Un grafico a linee che confronta la frazione di richieste HTTP/2 di prima parte con le richieste HTTP/2 o gQUIC di terze parti. Il grafico ordina i siti Web in base alla frazione delle richieste HTTP/2. Il 45% dei siti Web non ha richieste di prima parte HTTP/2. Oltre la metà dei siti web serve richieste di terze parti solo su HTTP/2 o gQUIC. L'80% dei siti Web ha il 76% o più richieste HTTP/2 o gQUIC di terze parti.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1409316276&format=interactive",
+  sql_file="http2_1st_party_vs_3rd_party.sql",
+  sheets_gid="733872185"
+)
+}}
+
+C'è qualche differenza nei tipi di contenuto serviti su HTTP/2 o gQUIC? La Figura 22.8 mostra, ad esempio, che il 90% dei siti Web serve il 100% di font e audio di terze parti su HTTP/2 o gQUIC, solo il 5% su HTTP/1.1 e il 5% sono un mix. La maggior parte delle risorse di terze parti sono script o immagini e vengono pubblicate esclusivamente su HTTP/2 o gQUIC rispettivamente sul 60% e 70% dei siti web.
+
+{{ figure_markup(
+  image="http2-third-party-http2-usage-by-content-type.png",
+  caption="Un grafico a barre che confronta la frazione di richieste HTTP/2 di terze parti in base al tipo di contenuto. Tutte le richieste di terze parti vengono servite su HTTP/2 o gQUIC per il 90% di audio e font, l'80% di css e video, il 70% di html, immagini e testo e il 60% di script.",
```

Comment:
Looks like you merged the caption and the description.

The caption is:
```
  caption="The fraction of known third-party HTTP/2 or gQUIC requests by content-type per website.",
```

I've auto translated it to the following:

```suggestion
  caption="La frazione di richieste HTTP / 2 o gQUIC di terze parti note per tipo di contenuto per sito web.",
  description="Un grafico a barre che confronta la frazione di richieste HTTP/2 di terze parti in base al tipo di contenuto. Tutte le richieste di terze parti vengono servite su HTTP/2 o gQUIC per il 90% di audio e font, l'80% di css e video, il 70% di html, immagini e testo e il 60% di script.",
```